docs/eventing/accessing-traces.md

```diff
@@ -0,0 +1,112 @@
+
+## Configuration
+
+Most Knative Eventing tracing configuration is handled by the `config-tracing` ConfigMap in the `knative-eventing` namespace. This single ConfigMap controls tracing for:
+ - Brokers
+ - Triggers
+ - Channel Dispatchers
+     - Channel Dispatchers are the name we use to describe the data plane portion of a Channel. When an event is sent to a Channel, the code handling that event is the Channel's dispatcher. Similarly, when an event is sent from a Channel to one of its Subscribers, the code handling sending that event is the Channel's dispatcher.
+
+Here is an example ConfigMap that samples 10% of all CloudEvents.
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-tracing
+  namespace: knative-eventing
+data:
+  enable: "true"
+  zipkin-endpoint: "http://zipkin.istio-system.svc.cluster.local:9411/api/v2/spans"
+  sample-rate: "0.1
+```
+
+To see your current configuration:
+
+```shell script
+kubectl -n knative-eventing get configmap config-tracing -oyaml
+```
+
+Configuration options:
+
+  * `backend`: Valid values are `zipkin`, `stackdriver`, or `none`. The default is `none`.
+
+ * `zipkin-endpoint`: Specifies the URL to the zipkin collector where you want to send the traces. Must be set if backend is set to `zipkin`.
+
+ * `stackdriver-project-id`: Specifies the GCP project into which stackdriver traces will be written. Only has an effect if the back is set to `stackdriver`. If unspecified, the project-id is read from GCP metadata when running on GCP.
+
+ * `sample-rate`: Valid values are one digit decimals from `0` to `1`. Specifies the percentage amount of all the requests that you want to trace.
+
+ * `debug`: Valid values are `true` or `false`. If not specified, defaults to `false`. Set to `true` to enable debug mode, which bypasses sampling and sends all spans to the server.
+
+Updating the ConfigMap will cause the new configuration to go live almost immediately:
+
+```shell script
+kubectl -n knative-eventing edit configmap config-tracing
+```
+
+### Importer Configuration
+
+Most importers do _not_ use the configuration from the shared ConfigMap. Instead, they have a static, 1% sampling rate.
+
+
+## Tools
+
+To access the traces, you will use a tool such as Zipkin or Jaeger. Follow the Knative Serving [instructions](./../serving/accessing-traces.md) for accessing the traces:
+ - [Zipkin](./../serving/accessing-traces.md#zipkin)
+ - [Jaeger](./../serving/accessing-traces.md#jaeger)
+
+### Example
+
+For this example, everything happens in the `default` namespace.
+- Broker named `default`
+- Two Triggers associated with the Broker:
+    - `mutator-t` - Filters to only allow events whose type is `original`. It will reply with an identical event, except the replied event's type will be `mutated`.
+    - `display-mutated` - Filters to only allow events whose type is `mutated`.
+- An event is sent in with the type `original`.
+
+So we expect the event to do the  following:
```

Comment:
Given the above example objects and configuration, the expected path and behavior of an event is as follows: